### PR TITLE
https://issues.redhat.com/browse/ACM-13244 missing host docs

### PR DESCRIPTION
**Pull request by oafischer**
2.11, 2.10, 2.9 and 2.8

### CODE REVIEW

**Review by CrystalChun — CHANGES_REQUESTED** [review]
Thanks for adding these changes in! The new content looks great, however, it's most relevant under this topic: https://github.com/stolostron/rhacm-docs/blob/2.10_stage/clusters/cluster_lifecycle/cim_auto_add_host.adoc since the converged flow affects hosts that are added automatically via BMH

**Comment by oafischer** [comment]
Moved the section and updated the PR @CrystalChun, thanks!

**Review by CrystalChun — APPROVED** [review]
/lgtm
Thanks Oliver!

File: clusters/cluster_lifecycle/cim_auto_add_host.adoc [hunk 1]
```diff
@@ -85,6 +85,34 @@ spec:
 
 After turning on the host, the image starts downloading. This might take a few minutes. When the host is discovered, an `Agent` custom resource is created automatically.
 
+[#auto-add-host-steps-converged]
+== Disabling converged flow
+
+Converged flow is enabled by default. If your hosts do not appear, you might need to temporarily disable converged flow. To disable converged flow, complete the following steps:
+
+. Create the following ConfigMap on the hub cluster:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cim_auto_add_host.adoc]:
```suggestion
. Create the following config map on the hub cluster:
```

File: clusters/cluster_lifecycle/cim_auto_add_host.adoc [hunk 2]
```diff
@@ -85,6 +85,34 @@ spec:
...
+----
++
+*Note:* When you set `ALLOW_CONVERGED_FLOW` to `"false"`, you also disable any features enabled by the Ironic Python Agent. 
+
+. Apply the ConfigMap by running the following command:
+
++
+----
+oc annotate --overwrite AgentServiceConfig agent unsupported.agent-install.openshift.io/assisted-service-configmap=my-assisted-service-config
+----
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cim_auto_add_host.adoc]:
```suggestion
+
[source,bash]
----
oc annotate --overwrite AgentServiceConfig agent unsupported.agent-install.openshift.io/assisted-service-configmap=my-assisted-service-config
----
```

File: clusters/cluster_lifecycle/cim_auto_add_host.adoc [hunk 3]
```diff
@@ -126,3 +154,5 @@ If you do not have a bare metal host on your hub cluster and want remove managed
 - See xref:../credentials/credential_on_prem.adoc#creating-a-credential-for-an-on-premises-environment[Creating a credential for an on-premises environment]
 
 - To learn more about scaling compute machines, see link:https://docs.openshift.com/container-platform/4.14/machine_management/manually-scaling-machineset.html[Manually scaling a compute machine set] in the {ocp-short} documentation.
+
+- To learn more about converged flow, see xref:../clusters/release_notes/known_issues.adoc#deploy-managed-stuck-pending[Managed cluster stuck in _Pending_ status after deployment].
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cim_auto_add_host.adoc]: This link is broken

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: clusters/cluster_lifecycle/cim_auto_add_host.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/cim_auto_add_host.adoc]: Should be fixed now, thanks for the review!

File: clusters/cluster_lifecycle/cim_auto_add_host.adoc [hunk 4]
```diff
@@ -85,6 +85,35 @@ spec:
 
 After turning on the host, the image starts downloading. This might take a few minutes. When the host is discovered, an `Agent` custom resource is created automatically.
 
+[#auto-add-host-steps-converged]
+== Disabling converged flow
+
+Converged flow is enabled by default. If your hosts do not appear, you might need to temporarily disable converged flow. To disable converged flow, complete the following steps:
+
+. Create the following config map on the hub cluster:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cim_auto_add_host.adoc]:
```suggestion
. Create the following config map on your hub cluster:
```

File: clusters/cluster_lifecycle/cim_auto_add_host.adoc [hunk 5]
```diff
@@ -85,6 +85,35 @@ spec:
...
+metadata:
+  name: my-assisted-service-config
+  namespace: multicluster-engine
+data:
+  ALLOW_CONVERGED_FLOW: "false"
+----
++
+*Note:* When you set `ALLOW_CONVERGED_FLOW` to `"false"`, you also disable any features enabled by the Ironic Python Agent. 
+
+. Apply the ConfigMap by running the following command:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cim_auto_add_host.adoc]:
```suggestion
. Apply the config map by running the following command:
```

**Review by dockerymick — APPROVED** [review]
2 more comments and the link works! Thanks Oliver. Besides my suggestions, this looks good to me. Approving!

**Pull request merged**